src/components/NewToDo/NewToDo.jsx

```diff
@@ -1,21 +1,19 @@
 import React, { useState } from "react";
 import { TextField } from "@mui/material";
 import "./NewToDo.css";
-import { useDispatch } from "react-redux";
-import { addToDo as reduxAddToDo } from "../../redux/toDoSlice/toDoSlice";
-import {v4 as uuidv4} from 'uuid'
+import { v4 as uuidv4 } from "uuid";
+import connectNewToDo from "../containers/NewToDoContainer";
```

Comment:
import connectNewToDo from "../containers/NewToDoContainer";
no longer need to import this

Reply:
deleted this line